src/Pretty.hs

```diff
@@ -284,6 +293,17 @@ instance Pretty VarInfo core => Pretty VarInfo (ScopedString core) where
 instance PrettyBinder VarInfo CompleteDecl where
   ppBind env (CompleteDecl d) = ppBind env d
 
+instance PrettyBinder VarInfo (Seq CompleteDecl) where
+  ppBind env decls = over _1 vsep
+                   $ foldr go (\e -> (mempty, e)) decls mempty
+    where
+      go :: CompleteDecl
+         -> (Env Var () -> ([Doc VarInfo], Env Var ()))
+         -> (Env Var () -> ([Doc VarInfo], Env Var ()))
+      go decl cc e = let (doc, e') = ppBind (env <> e) decl
+                         (docs, e'') = cc (e <> e')
+                     in (doc:docs, e'')
+
 instance PrettyBinder VarInfo [CompleteDecl] where
```

Comment:
Is the list instance still used?

Reply:
no i accidentally left this in, removed in https://github.com/gelisam/klister/pull/215/commits/d04248827f8d6e931c8ddcb4c23d6075cd93574c